yw_website/yw_website/apps/website/static/design/detailed_design.css

```diff
@@ -86,3 +71,97 @@
     width: 100%;
     height: 5em;
 }
+
+
+.delete, .modal-close, .is-unselectable, .button, .file, .breadcrumb, .pagination-previous,
```

Comment:
if we're copy and pasting css, we should indicate where the source is

Reply:
done